docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: AWS Console and CLI access
+description: How to access AWS with Teleport.
+labels:
+ - how-to
+ - zero-trust
+---
+
+You can protect the AWS Management Console and AWS APIs with Teleport.
+This allows you to manage access to AWS infrastructure with Teleport features like Access Requests, Access Lists, and identity locking.
+You can also configure Teleport to provide different levels of AWS access automatically when users authenticate using your single sign-on solution.
+
+<Admonition type="tip">
+When compared with the [AWS Console and CLI access via Teleport Agent](aws-console.mdx) method,
+this guide does not require a Teleport Agent to be installed and provides 100% compatibility with AWS SDK based tools (e.g., terraform).
+However, it does not record the requests in Teleport's audit logs, only that a session was started by a given user.
+</Admonition>
```

Comment:
I think we need to lead with this information right in the introduction.

Basically say there are two options for accessing AWS with Teleport, briefly explain the difference between them, tell the reader they are currently viewing the guide for option 2, and link them to option 1 for futher reading.